src/main/scala/pirate/Read.scala

```diff
@@ -196,6 +196,7 @@ case object ReadErrorEmpty extends ReadError
 case object ReadErrorNotEnoughInput extends ReadError
 case object ReadErrorTooMuchInput extends ReadError
 case class ShowHelpText(sub: Option[String]) extends ReadError
+case class ShowVersion(version: String) extends ReadError
```

Comment:
Yeah, it's a bit ugly, there could be a ReadError which lifts up a ParseError. But for now I think this should be fine.